<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QWidget" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>73</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Let's sign up</string>
   </property>
  </widget>
  <widget class="WCommonEdit" name="userNameEdit" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>125</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WCommonEdit" name="passwordEdit" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>197</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WCommonEdit" name="passwordEdit2" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>269</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WButton" name="signBtn" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>637</y>
     <width>544</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="WMobileLogin" name="verfyCodeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>341</y>
     <width>640</width>
     <height>285</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WCommonEdit</class>
   <extends>QWidget</extends>
   <header location="global">wcommonedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WButton</class>
   <extends>QWidget</extends>
   <header location="global">wbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WMobileLogin</class>
   <extends>QWidget</extends>
   <header location="global">wmobilelogin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
